src/features/auth/styles/auth.ts

```diff
@@ -66,7 +63,10 @@ export const styles = StyleSheet.create({
   },
   inputBlock: {
     paddingVertical: 20,
-    width: width - paddingInputBlock,
+    width: width - 100,
```

Comment:
@DiamondRock597 Linter not allow magic numbers, you can use const value paddingOffset = 100;
And use paddingOffset whenever you need.